src/qt/transactionoverviewwidget.h

```diff
@@ -5,11 +5,8 @@
 #ifndef BITCOIN_QT_TRANSACTIONOVERVIEWWIDGET_H
 #define BITCOIN_QT_TRANSACTIONOVERVIEWWIDGET_H
 
-#include <qt/transactiontablemodel.h>
-
 #include <QListView>
```

Comment:
`QListView` is being included in both files now. Could remove it from here.

Reply:
From our [Developer Notes](https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#source-code-organization):
> Every `.cpp` and `.h` file should `#include` every header file it directly uses classes, functions or other definitions from, even if those headers are already included indirectly through other headers.